Editor/Window/ManagedEC2Page.cs

```diff
@@ -0,0 +1,145 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using AmazonGameLift.Editor;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.UIElements;
+using OperatingSystem = Amazon.GameLift.OperatingSystem;
+
+namespace AmazonGameLift.Editor
+{
+    public class ManagedEC2Page
+    {
+        private readonly VisualElement _container;
+        private readonly DeploymentSettings _model;
+        private readonly Button _deployButton;
+        private readonly Button _redeployButton;
+        private readonly Button _deleteButton;
+        private readonly Button _launchClientButton;
+        private readonly DeploymentScenariosInput _deploymentScenariosInput;
+        private readonly FleetParametersInput _fleetParamsInput;
+        private readonly Label _ec2DeploymentStatusLabel;
+
+        public ManagedEC2Page(VisualElement container)
+        {
+            _container = container;
+            _model = DeploymentSettingsFactory.Create();
```

Comment:
Please rename this to `_deploymentSettings`, it's not evident what _model represents without referring back to this ine

Reply:
renamed